configure/builder.go

```diff
@@ -0,0 +1,63 @@
+package configure
+
+import (
+	eh "github.com/looplab/eventhorizon"
+)
+
+// CommandHandlerMiddleware for supporting middleware registration
+type CommandHandlerMiddleware func(eh.CommandHandler) eh.CommandHandler
```

Comment:
Might want to move this to a better location

Reply:
Yes, here is my plan on where to add that interface: https://github.com/maxekman/eventhorizon/commit/c8bf731bd559365d78b1767ec3a6c519922d5e2d